frontend/src/js/components/modals/CachedFiles/Tabs/FileTab.jsx

```diff
@@ -336,6 +337,11 @@ class FileTab extends Component {
         if (this.state.selectedFile) {
             selected_file_path = cleanPath(this.state.selectedFile.path) + "/" + this.state.selectedFile.name;
         }
+        if(this.props.hasError){
+            this.props.handleError();
```

Comment:
This should be added to proptypes and labeled as required. If it isn't required, you could either check that it exists before executing it, or adding some sort of default prop.